### DISCUSSION TITLE
Cannot generate properly "hexo g" just after the installtion ("hexo s" works fine)

### DISCUSSION THREAD

**Discussion by junyiphys** [hexojs/hexo#5103] [Question / Answer] [answered]
Hi all, 

I used to build the blog through hexo on Fedora, but this time, I tried to install hexo in win11 (using nvs for installing node js (v. 18.12.1)) and the latest git version. After I followed the tutorial steps to generate the file, I could not get the proper landscape themes as the official site demonstrates. In short, the blog can show appropriately with "hexo server" but cannot display properly with the "index.html" from the file "hexo g." Please let me know if I didn't do something wrong or need to provide further info for debugging this prob.

Noted: no errors and just native env in config.yml.

Here's the detailed process of my installation:

1. Git installation from https://git-scm.com/download/win _(ver. 2.38.1)_ and nvs installation with nodejs _18.12.1_ (I also tried to install v13 and v16, but didn't work at all)
2. Install Hexo with the command npm install -g hexo-cli  _(ver.6.3)_
3. Create the hexo blogsite
```
$ hexo init <folder>
$ cd <folder>
$ npm install
```
4. Create a testing Hexo frame through hexo g and then hexo s. <- I got a problem here. hexo g and hexo s show a different result. 

**hexo -v**
hexo: 6.3.0
hexo-cli: 4.3.0
os: win32 10.0.22000 
node: 18.12.1
v8: 10.2.154.15-node.12
uv: 1.43.0
zlib: 1.2.11
brotli: 1.0.9
ares: 1.18.1
modules: 108
nghttp2: 1.47.0
napi: 8
llhttp: 6.0.10
openssl: 3.0.7+quic
cldr: 41.0
icu: 71.1
tz: 2022b
unicode: 14.0
ngtcp2: 0.8.1
nghttp3: 0.7.0

![image](https://user-images.githubusercontent.com/26758810/201524760-7edc55ea-776d-43f1-8d2c-de0f8b376d64.png)

**Comment by stevenjoezhang** [comment, marked as answer]
If you are double-clicking to open the `index.html`, then the css will not load because it's using file protocol. You need to use a web server to serve these pages, for example
```
python -m http.server
```

**Reply by junyiphys** [reply]
thx for the suggestion! But it is still weird that even if I deploy to GitHub, the CSS still doesn't work :(( (Computer and mobile both don't work properly) Is that still the same issue?
https://zazatw.github.io/blog/

But my original one works fine (uploaded through fedora about an ago lol) https://zazatw.github.io/cblog/)

btw, here's the error I got after I typed "python3 -m http.server" to monitor the file while opening

![image](https://user-images.githubusercontent.com/26758810/201527924-b4fb1b4a-b110-47d5-b9ed-abccdebd2f51.png)

**Reply by junyiphys** [reply]
The uploaded blog with the correct theme was fixed by an unknown reason. It seems like I should wait more time (30 mins?) to get the correct CSS to render on the hexo blog at the beginning.

**Reply by stevenjoezhang** [reply]
github pages may take some time to refresh the cache. You can also try other static blog hosting services, such as Cloudflare Pages or Netlify
